pkg/controllers/managementuser/rbac/handler_base.go

```diff
@@ -129,6 +129,13 @@ func Register(ctx context.Context, workload *config.UserContext) {
 	management.Management.RoleTemplates("").AddHandler(ctx, "cluster-roletemplate-sync", newRTLifecycle(r))
 	relatedresource.WatchClusterScoped(ctx, "enqueue-beneficiary-roletemplates", newRTEnqueueFunc(rtInformer.GetIndexer()),
 		management.Wrangler.Mgmt.RoleTemplate(), management.Wrangler.Mgmt.RoleTemplate())
+
+	management.Wrangler.Mgmt.ProjectRoleTemplateBinding().Cache().AddIndexer(namespace.PrtbByRoleTemplateIndex, namespace.PrtbByRoleTemplateName)
+	nsEnqueuer := namespace.NsEnqueuer{
+		PrtbCache: management.Wrangler.Mgmt.ProjectRoleTemplateBinding().Cache(),
+		NsIndexer: nsInformer.GetIndexer(),
```

Comment:
I'd prefer to use a wrangler cache, but we already have a Norman one in place so I re-used it instead of maintaining 2 caches for the same objects.